src/_cffi_src/build_openssl.py

```diff
@@ -38,8 +38,15 @@ def _osx_libraries(build_static):
 
 
 _PRE_INCLUDE = """
+#include <openssl/opensslv.h>
+/*
+    LibreSSL removed e_os2.h from the public headers so we'll only include it
+    if we're using vanilla OpenSSL.
+*/
```

Comment:
Nope, `_WIN32` is guaranteed defined on 64-bit platforms as well. https://msdn.microsoft.com/en-us/library/b0084kay.aspx